examples/hello_world_plugin/HelloWorldPlugin.cc

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (C) 2020 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#include <ignition/common/SingletonT.hh>
+
+#include <ignition/plugin/Register.hh>
+
+#include "ignition/rendering/RenderEnginePlugin.hh"
+#include "ignition/rendering/base/BaseRenderEngine.hh"
+#include "ignition/rendering/base/BaseRenderTypes.hh"
+
+namespace mock
+{
+  class HelloWorldRenderEngine :
```

Comment:
Can you add a `\brief` doxygen comment for this and the `HelloWorldPlugin` class?

Reply:
0dd768276f5742bd222ca33340f1035a413542aa